code/collections.rb

```diff
@@ -1,15 +1,18 @@
 # collections
 
 # 1. Create a variable `point1` and assign it a hash representing a coordinate with keys `:x` and `:y` which have integer values
-
+point1={
+    x: 25,
+    y: 32
+}
 # 2. Print the y value
-
+point1[:y]
```

Comment:
👍  you got the right value, but we also wanted you to use `puts` to print it out:

`puts point1[:y]`